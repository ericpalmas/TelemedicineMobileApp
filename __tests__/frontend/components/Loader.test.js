import 'react-native';
import React from 'react';
import Loader from '../../../frontend/components/Loader'
import renderer from 'react-test-renderer';


it('renders correctly', () => {
  const tree = renderer.create(<Loader />).toJSON();
  expect(tree).toMatchSnapshot();
});