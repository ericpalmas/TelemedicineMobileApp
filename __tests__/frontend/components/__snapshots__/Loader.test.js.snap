// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "flex": 1,
        "justifyContent": "center",
      },
      Object {
        "flexDirection": "row",
        "justifyContent": "space-around",
        "padding": 10,
      },
    ]
  }
>
  <ActivityIndicator
    color="#00ff00"
    size="large"
  />
</View>
`;
